lib/Command/ActiveCalls.php

```diff
@@ -84,6 +106,48 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 			$data = ['calls' => $numCalls, 'participants' => $numParticipants];
 			$this->writeArrayInOutputFormat($input, $output, $data);
 		}
-		return 1;
+		return $input->getOption('non-zero') ? 1 : 0;
+	}
+
+	protected function executeDetails(InputInterface $input, OutputInterface $output): int {
+		$query = $this->connection->getQueryBuilder();
```

Comment:
Wouldn't it be better in a Service to make this query usable by other parts of code?

Reply:
I had the same thought and then i wondered if I even want this code to be available for reuse as it somewhat undermins the privacy.
For now I would leave it out and only move it once needed

Reply:
Will be easier to found this query, maybe in the `SessionService`, to reuse in other places.

Also, if we isolate this query, it's makes easier to write unit tests because now we only have integration tests covering Talk commands. Should be good to have more unit tests covering more scenarios than integration tests.